### ISSUE TITLE
Azure -functions - AddJosnFile 

### ISSUE THREAD

**Issue by joseclament** [Azure/azure-functions-host#6928]
Hi,
I am trying to load appsettings.json file using function app V1, for loading various function configuration. But it's failing with the below exception.

Could not load file or assembly 'System.Runtime.CompilerServices.Unsafe, Version=4.0.4.1, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies. The system cannot find the file specified.
Code snipset

```csharp
        static Function1()
        {
            var currentDirectory = @"D:\home\site\wwwroot\";
            bool isProduction = Directory.Exists(currentDirectory);
            if (!isProduction)
            {
                currentDirectory = Environment.CurrentDirectory;
            }    

            var appSettingsFileName = currentDirectory + @"\appsettings.json";
            var builder = new ConfigurationBuilder()
                .SetBasePath(currentDirectory)
                .AddJsonFile(appSettingsFileName , optional: false, reloadOnChange: true)               
                .AddEnvironmentVariables();
          
            var configBuilder = builder.Build();           
            var customConfig = new CustomConfig();
            configBuilder.GetSection("CustomConfig").Bind(customConfig);

            var mailSettings = new MailSettings();
            configBuilder.GetSection("MailSettings").Bind(mailSettings);

            var myKeyPairValue1 = configBuilder.GetSection("MyKeyPairValue1String");
            var myKeyPairValue2 = configBuilder.GetSection("MyKeyPairValue2Number");
        }

        [FunctionName("Function1")]
        public static void Run([TimerTrigger("0 */1 * * * *")] TimerInfo myTimer, TraceWriter log, ExecutionContext execution)
        {
            var j = execution.FunctionDirectory;
            var m = execution.FunctionDirectory;
            log.Info($"C# Timer trigger function executed at: {DateTime.Now}");
        }
    }
```
![image](https://user-images.githubusercontent.com/35963449/99922145-c7430f00-2d26-11eb-8bf6-9e41a912969e.png)

Can I achieve this using function app V1 (.Net 4.8version)?

**Comment by joseclament** [comment]
Hi,
I have found that the "appsettings.json" file couldn't be found by the function exe.
Got the below error details. But the previous error was misleading as it was throwing a different error message in the function app console.

Exception thrown: 'System.IO.FileNotFoundException' in Microsoft.Extensions.Configuration.FileExtensions.dll
Exception thrown: 'System.IO.FileNotFoundException' in mscorlib.dll
Exception thrown: 'System.IO.FileNotFoundException' in mscorlib.dll
Exception thrown: 'System.IO.FileNotFoundException' in mscorlib.dll
Exception thrown: 'System.IO.FileNotFoundException' in mscorlib.dll

I have updated the code to see if the "appsetting.json" there in the output folder and it's there.(FunctionApp1\bin\Debug\net48), but one level above the bin folder where all the function app dlls were reside.

Does  the file should be in the bin folder (FunctionApp1\bin\Debug\net48\bin)?
